directory/cli/src/test_utils.py

```diff
@@ -62,6 +63,16 @@ def _reload_with_advanced_set_to(val):
     os.environ['ADVANCED'] = val
     importlib.reload(directory)
 
+def mock_options_passed_to_ipa(mocker, directory_command, ipa_run_arguments, input_stream=None):
```

Comment:
This function now appears twice identically in this file (see https://github.com/alces-software/userware/pull/74/files#diff-c2a87f9cbe4a4e11aeb63e254411ecc7R19) - possibly due to a mistake in the merging when rebasing?

Reply:
Yep! I was only paying attention to the marked areas that the rebase was asking to be looked at. I should have double checked the whole file, hopefully there isn't too much of this present.

Reply:
No worries :slightly_smiling_face:. I didn't see anything else but just happened to notice that I thought I'd already merged a PR with this in, so could be worth a skim to check.